mixins/eirini/templates/configmap.yaml

```diff
@@ -95,4 +95,9 @@ data:
     eirini_cert_path: "/etc/eirini/secrets/eirini-client.crt"
     eirini_key_path: "/etc/eirini/secrets/eirini-client.key"
     ca_path: "/etc/eirini/secrets/eirini-client.ca"
+  instance-index-env-injector.yml: |
+    service_name: eirinix-instance-index-env-injector
+    service_port: 8443
+    workload_namespace: {{ .Values.eirini.opi.namespace }}
+    service_namespace: {{ .Release.Namespace }}
```

Comment:
This change is adding the env injector webhook from the Eirini code (https://github.com/cloudfoundry-incubator/eirini/tree/master/cmd/instance-index-env-injector). It makes sure, the application pods get the CF_INSTANCE_INDEX env variable.